tiatoolbox/dataloader/slide_info.py

```diff
@@ -34,16 +35,18 @@ def slide_info(input_path, output_dir=None):
 
     input_dir, file_name = os.path.split(input_path)
 
-    print(file_name, flush=True)
+    if verbose:
+        print(file_name, flush=True)
     _, file_type = os.path.splitext(file_name)
 
     if file_type in (".svs", ".ndpi", ".mrxs"):
-        wsi_reader = wsireader.WSIReader(
+        wsi_reader = wsireader.OpenSlideWSIReader(
             input_dir=input_dir, file_name=file_name, output_dir=output_dir
         )
-        info = wsi_reader.slide_info()
+        info = wsi_reader.slide_info
+        if verbose:
+            print(info)
     else:
-        print("File type not supported")
-        info = None
+        raise FileNotSupported
```

Comment:
Doing it this way will raise a generic error - do we not want to also give the filename for which it failed on?

Reply:
Updated it in this commit https://github.com/TIA-Lab/tiatoolbox/commit/3ac21d1b573dc9f272a8527ef14e866b52c9c631